app/scripts/controllers/provisionedService.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:ProvisionedServiceController
+ * @description
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('ProvisionedServiceController', function ($scope,
+                                             $routeParams,
+                                             DataService,
+                                             ProjectsService,
+                                             $filter) {
+    $scope.projectName = $routeParams.project;
```

Comment:
It isn't, there are a few bits here and there like that where wasn't sure about them keeping around... part of my questions that need to filter these bits out